web/modules/custom/dpl_react_apps/src/Controller/DplReactAppsController.php

```diff
@@ -202,7 +202,7 @@ public function work(string $wid): array {
         'genre-and-form-text' => $this->t('Genre', [], $c),
         'get-online-text' => $this->t('Get online', [], $c),
         'go-to-e-reolen-text' => $this->t('Go to e-Reolen', [], $c),
-        'go-to-text' => $this->t('Go to', [], $c),
+        'go-to-text' => $this->t('Go to @source', [], $c),
```

Comment:
Please consider if the text right above this change specifically for eReolen is still relevant now that we have placeholders.

Reply:
Great point. I will remove it from dpl-react too to avoid confusion.